src/config/entities/__tests__/configuration.ts

```diff
@@ -33,6 +34,27 @@ export default (): ReturnType<typeof configuration> => ({
     level: 'debug',
     silent: process.env.LOG_SILENT?.toLowerCase() === 'true',
   },
+  prices: {
+    baseUri:
+      process.env.PRICES_PROVIDER_API_BASE_URI ||
+      'https://api.coingecko.com/api/v3',
+    apiKey: process.env.PRICES_PROVIDER_API_KEY,
+    pricesTtlSeconds: parseInt(process.env.PRICES_TTL_SECONDS ?? `${300}`),
+    chains: {
```

Comment:
This Coingecko-specific chain configuration data could be externalized to the Safe Config Service. However, this can be done in a separate PR, because it also depends on whether we plan to continue to use Coingecko for the long term.